tests/contrib/tornado/test_config.py

```diff
@@ -35,3 +35,24 @@ def test_tracer_is_properly_configured(self):
         assert self.tracer.writer._filters is not None
         assert len(self.tracer.writer._filters) == 1
         assert isinstance(self.tracer.writer._filters[0], FilterRequestsOnUrl)
+
+
+class TestTornadoDefaultService(TornadoTestCase):
+    """ Ensure that the default service for a Tornado web application is configured
+    """
+    def get_settings(self):
+        return {
+            'datadog_trace': {
+                'default_service': 'custom-tornado',
+            },
+        }
+
+    def test_service(self):
```

Comment:
any reason to have this be it's own test class as opposed to collapsing into the above one?

Reply:
This is because the above test class disables the tracer.